README.md

```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
 
 [![Build Status](https://circleci.com/gh/tikv/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/tikv/tikv) [![Coverage Status](https://codecov.io/gh/tikv/tikv/branch/master/graph/badge.svg)](https://codecov.io/gh/tikv/tikv) ![GitHub release](https://img.shields.io/github/release/tikv/tikv.svg)
 
+
+TiKV is an open source distributed Key-Value storage system; Unlike other traditional NoSQL systems, TiKV not only provides classical Key-Value style APIs, but also ACID semantic APIs.
```

Comment:
seem these are duplicated with following introductions.

Reply:
The reason why I put a short description here is I think maybe some engineers can't have the patience to read the full description below, so a '1-sentence' intro seems a good idea.

Reply:
I agree with @c4pt0r that 1-sentence intro is a good idea, but let's avoid duplicating information. Let's edit and move [the introduction](https://github.com/tikv/tikv/pull/3795#diff-04c6e90faac2675aa89e2176d2eec7d8R15) here.

Reply:
TiKV ("Ti" stands for Titanium) is an open source distributed transactional key-value database. Unlike other tranditional NoSQL systems, TiKV not only provides classical Key-Value APIs, but also transactional APIs with ACID compliance. Built in Rust and powered by Rust, TiKV was originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.